indra/llfilesystem/lldir.cpp

```diff
@@ -864,6 +864,25 @@ std::vector<std::string> LLDir::findSkinnedFilenames(const std::string& subdir,
     return results;
 }
 
+std::string LLDir::getFileContents(const std::string& filename) const
```

Comment:
This is a very confusing name and in a confusing place. How is content of LLFile is related to LLDir?
Suggestion: some variant of LLFile::fileAsString(filename)?

Reply:
Also could have easily been static instead of needing gDirUtilp instance.